Week-1/Assignment-2/readme.md

```diff
@@ -0,0 +1,44 @@
+<h1>Exercise 1.2 :</h1>
```

Comment:
Why this readme?

Reply:
Readme to Show Question and what to do. For Personal and others Ease.